index.html

```diff
@@ -1,25 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
   <link rel="stylesheet" href="style.css">
   <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap"
+    rel="stylesheet">
 </head>
+
 <body>
   <!-- Nav Bar menu on mobile -->
   <header>
     <nav class="mobile-nav-bar">
       <a class="intro-section__headtitle" href="">My Logo</a>
-      <img class="hamburguer-menu" src="Assets/Unionhamb.png" alt="mobile menu">
+      <div class="hamburguer-menu " alt="mobile menu" 
+      onclick="menuOpen()"><img src="Assets/Unionhamb.png"></div>
+      <div id="menu" class="menu_basic">
+        <div class="menu-items">
+          <div class="close__btn close-button-mobile mobile-close close" onclick="menuClose()"><img src="Assets/Vectorwhite-x.png"></div>
+          <a href="#portfolio" onclick="menuClose()" class="mobile-menu-items">Portfolio</a>
+          <a class="mobile-menu-items" href="#about" onclick="menuClose()">About</a>
+          <a class="mobile-menu-items" href="#Contact" onclick="menuClose()">Contact</a>
+        </div>
+      </div>
```

Comment:
- Well done 👍 on implementing the mobile menu. It is working perfectly fine. But it needs some design modification as per the project [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=533fhzG3siX8egvo-0). I am addressing those below. Kindly fix those issue.

- [ ] After clicking the `hamburger menu`, a popup window comes. In the right side of the popup window I can see a `X` to close the popup window. But in the background of the `X` I can still see the `hamburger menu`. It shouldn't be there.
- ![image](https://github.com/cancelei/Portfolio-Option-2/assets/60083437/e29abb7a-8ba0-46b5-a1c3-26f0f9ce0eb1)

- [ ] There are too many `gaps` between the portfolio, about, and contact menu. It is not matching the given design.
- ![image](https://github.com/cancelei/Portfolio-Option-2/assets/60083437/ab634779-18bf-4a4d-af11-79c0baa5482f)

- [ ] In the given [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-546&t=533fhzG3siX8egvo-0) I can see the `background color` of the menu is `blur`. But your design is too much transparent, I can easily read the text outside the menu, and it confuses the viewer which option is menu and which are other texts.
- Given Design:
- ![image](https://github.com/cancelei/Portfolio-Option-2/assets/60083437/edc1ed41-0489-452f-a998-34f6cda6616a)
- Your Design:
- ![image](https://github.com/cancelei/Portfolio-Option-2/assets/60083437/c1cf006c-5807-486d-b6ac-dfafd56144ec)

Reply:
![current](https://github.com/cancelei/Portfolio-Option-2/assets/96393104/54061e27-659f-432e-995a-49f39e9b0f8a)

@Tufahel I tried to fix the background, but the items seem to be displaced. When I blur them it blurs the mobile menu items. 

Since I have no mentor this week, could you give me some assistance in how to solve that? I experimented with positioning the elements, but it makes them displace and also don't solve the issue. 

Let me now if you can give me a hand on that.